lib/optimizely/helpers/validator.rb

```diff
@@ -112,9 +112,7 @@ def inputs_valid?(variables, logger = NoOpLogger.new, level = Logger::ERROR)
           next if value.is_a?(String) && !value.empty?
 
           is_valid = false
-          if logger_valid?(logger) && level
-            logger.log(level, "#{Optimizely::Helpers::Constants::INPUT_VARIABLES[key.to_s.upcase]} is invalid")
-          end
+          logger.log(level, "#{Optimizely::Helpers::Constants::INPUT_VARIABLES[key.to_s.upcase]} is invalid") if logger_valid?(logger) && level
```

Comment:
Hmm, I find this is harder to read than before, is that really one of our int rules? If so we should change it so that it won't complain about this. @apanzerj what are your thoughts on this kind of styling rule?

Reply:
Plz review the addressed change, it's also complying with rubocop rules.